AEPCore/Sources/eventhub/EventHub.swift

```diff
@@ -239,12 +244,11 @@ final class EventHub {
             }
         }
 
-        // TODO: Determine which version of Core to use in the top level version field
-        let data: [String: Any] = [EventHubConstants.EventDataKeys.VERSION: ConfigurationConstants.EXTENSION_VERSION,
+        let data: [String: Any] = [EventHubConstants.EventDataKeys.VERSION: EventHubConstants.VERSION_NUMBER,
```

Comment:
updated to version number in eventhub constants instead of configuration constants